administrator/language/de-DE/plg_system_schemaorg.ini

```
@@ -15,7 +15,7 @@ PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_DESCRIPTION_NOT_CONFIGURATED="Um die schema.or
 PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_DESCRIPTION_NOT_CONFIGURATED_ADMIN="Um die schema.org-Funktionalität zu nutzen, muss zunächst das Plugin konfiguriert werden. Auf <a href='index.php?option=com_plugins&amp;task=plugin.edit&amp;extension_id=%s' target='_blank'>diesen Link klicken, um das Plugin zu öffnen</a>, zu konfigurieren und zu speichern."
 PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_DESCRIPTION_NOT_CONFIGURED="Um die schema.org-Funktionalität zu nutzen, muss zunächst das Plugin durch den Administrator der Seite konfiguriert werden."
 PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_DESCRIPTION_NOT_CONFIGURED_ADMIN="Um die schema.org-Funktionalität zu nutzen, muss zunächst das Plugin konfiguriert werden. Auf <a href='index.php?option=com_plugins&amp;task=plugin.edit&amp;extension_id=%s' target='_blank'>diesen Link klicken, um das Plugin zu öffnen</a>, zu konfigurieren und zu speichern."
-PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_EXTEND_JED_DESC="<strong>Weitere Schema-Typen erforderlich?</strong> Mit den <a href='https://extensions.joomla.org/category/core-enhancements/schema-plugins' target='_blank' rel='noopener noreferrer'>Schema-Plugins aus dem Joomla! Extensions Directory (JED)</a> erweitern."
+PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_EXTEND_JED_DESC="<strong>Weitere Schema-Typen erforderlich?</strong> Mit den <a href='https://extensions.joomla.org/category/core-enhancements/schema-plugins' target='_blank' rel='noopener noreferrer'>Schema-Plugins aus dem Joomla-Extensions-Directory (JED)</a> erweitern."
```

```suggestion
PLG_SYSTEM_SCHEMAORG_FIELD_SCHEMA_EXTEND_JED_DESC="<strong>Weitere Schema-Typen erforderlich?</strong> Mit den <a href='https://extensions.joomla.org/category/core-enhancements/schema-plugins' target='_blank' rel='noopener noreferrer'>Schema-Plugins aus dem Joomla! Extensions Directory (JED)</a> erweitern."
```

Das müsste ein Eigennahme sein.